drivers/mpu9250/MPU9250.cpp

```diff
@@ -490,21 +399,23 @@ void MPU9250::_measure()
 		report->gyro_y = swap16(report->gyro_y);
 		report->gyro_z = swap16(report->gyro_z);
 
+
 		// Check if the full accel range of the accel has been used. If this occurs, it is
 		// either a spike due to a crash/landing or a sign that the vibrations levels
 		// measured are excessive.
-		if (report->accel_x == INT16_MIN || report->accel_x == INT16_MAX ||
-		    report->accel_y == INT16_MIN || report->accel_y == INT16_MAX ||
-		    report->accel_z == INT16_MIN || report->accel_z == INT16_MAX) {
+		if (report->accel_x == INT16_MIN || report->accel_x == INT16_MAX
```

Comment:
Got it.  I tried to fix the white space on these lines, hopefully restoring them to their original form.